src/Generator.jl

```diff
@@ -25,23 +25,44 @@ end
 """
 $(SIGNATURES)
 
+Attempts to append to a file at `\$(root)/\$(filename)`. `f` will be called with
+file stream (see [`open`](http://docs.julialang.org/en/latest/stdlib/io-network/#Base.open)).
+"""
+function appendfile(f, root, filename)
+    filepath = joinpath(root, filename)
+    info("Appending to $filename at $filepath")
+    open(f,filepath,"a")
+end
+
+"""
+$(SIGNATURES)
+
 Contents of the default `make.jl` file.
 """
-function make(pkgname)
+function make(pkgname, user)
     """
     using Documenter
-    using $(pkgname)
+    using $pkgname
 
     makedocs(
-        modules = [$(pkgname)]
+        modules = [$pkgname],
+        format = :html,
+        sitename = "$pkgname.jl",
+        pages = Any["Home" => "index.md"],
+        strict = true
     )
 
-    # Documenter can also automatically deploy documentation to gh-pages.
-    # See "Hosting Documentation" and deploydocs() in the Documenter manual
-    # for more information.
-    #=deploydocs(
-        repo = "<repository url>"
-    )=#
+    # for successful deployment, make sure to
+    # - add a gh-pages branch on github
+    # - run `import Documenter; Documenter.Travis.genkeys("$pkgname")` in a
+    #       *REPL* and follow instructions. For Windows, run from inside
+    #       git-bash.
```

Comment:
Should just provide a link to the main docs here, and just expand those if necessary.